ecs-init/docker/docker.go

```diff
@@ -423,5 +423,12 @@ func (c *Client) StopAgent() error {
 		return nil
 	}
 	stopContainerTimeoutSeconds := uint(10)
-	return c.docker.StopContainer(id, stopContainerTimeoutSeconds)
+	err = c.docker.StopContainer(id, stopContainerTimeoutSeconds)
+	if err != nil {
+		if _, ok := err.(*godocker.ContainerNotRunning); ok {
+			log.Info("Agent is not running")
```

Comment:
in this case we should consider phrasing log entry differently to suggest that it's not an error case - if we just read "Agent is not running" in the log, that sounds bad. Something along the lines of "Agent is already stopped" or "Agent stopped successfully"?

Reply:
yeah, changed to "Agent is already stopped"